property/views.py

```diff
@@ -516,6 +516,7 @@ def get_queryset(self):
             last_viewed__date__gte=date,
             is_published=True,
             is_sold=False,
+            is_deleted=False,
```

Comment:
What do you think of using the same implementation that viewing the list of properties uses

https://github.com/landvilleng/landville-backend-web-api/blob/75eb52bb3163af99d680caf56555974e635fedcd/property/views.py#L167

So that our implementation has

```py
Property.active_objects.all_published().filter(...our filtration criteria)
```

My reason is `all_published` makes a call to `_active` which just filters out the deleted properties

![image](https://user-images.githubusercontent.com/12128153/64631357-9b788980-d3ff-11e9-8079-cc341fcf37de.png)

There's also uniformity when you use the above suggestion, since it's still being used in another view

Reply:
@neelxie action on @Tevinthuku feedback and then we should be good to go.

Reply:
@Tevinthuku @alexxsanya this has been worked on.